charts/podinfo/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-version: 5.2.0
+version: 5.2.1
```

Comment:
Please undo this, the version is bump separately in the release PR.

Reply:
Ah ok sure. Done via 8979ed88793699dfe80e751de4385d61e4cd10bc